feature/bgp/bgp_isis_redistribution/otg_tests/bgp_isis_redistribution_test/bgp_isis_redistribution_test.go

```diff
@@ -316,8 +323,6 @@ func nonMatchingPrefixRoutePolicy(t *testing.T, dut *ondatra.DUTDevice) {
 	stmt.GetOrCreateConditions().GetOrCreateMatchPrefixSet().SetPrefixSet(v4PrefixSet)
 	gnmi.Update(t, dut, gnmi.OC().RoutingPolicy().Config(), rp)
 
-	// enable bgp isis redistribution
-	bgpISISRedistribution(t, dut)
```

Comment:
In func bgpISISRedistribution ,table connection import policy 'v4RoutePolicy' is set.
 
 The seventh testcase with name NonMatchingCommunityV6, does gnmi Replace of policy hierarchy in function nonMatchingCommunityRoutePolicyV6 .This removes v4RoutePolicy referred in bgpISISRedistribution.
 
 Hence to avoid failures related to replacing the policy without deleting the policy reference, these script modifications were done.

Reply:
Currently, bgpISISRedistribution is configured in 1st testcase and that v4 table connection config will remain for all subsequent testcases. Similarly for v6, bgpISISRedistributionV6 is configured in 5th testcase and config will remain.

v4RoutePolicy/v6RoutePolicy policy used in table connection import policy config 
is modified in each testcase as per requirement to configure a matching/non-matching prefix or community 
and checks if route is redistributed or not according to that.

In proposed change, script is modified to add/set the table connection config before each testcase and delete those at the end of each (v4 - bgpISISRedistribution/v6 - bgpISISRedistributionV6)